CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # 0.19.0
 
-<<<<<<< HEAD
-* Reworked enumerations / the EnumerationSchema to eliminate a OOM pitfall and improve ergonomics of SchemaVisitor
-=======
+## Documentation fix
+
+Prevent documentation from being generated for case class when the field are not generated because they're annotated with `@streaming`
```

Comment:
Instead of removing it altogether, could we just make a separate paragraph out of it in the docs of the holding structure ?